lib/site-utils.js

```diff
@@ -63,6 +64,8 @@ const EURO_SYMBOL_PREFIX = '€';
 const YEN_TEXT_PREFIX = '円';
 const YEN_SYMBOL_PREFIX = '￥';
 const GBP_SYMBOL_PREFIX = '£';
+const GBP_TEXT_PREFIX = 'gbp';
+const INR_SYMBOL_PREFIX = 'r ';
```

Comment:
Added space in the end, because price will be retrieved as R 9,444. If we remove space, it will not only break test case, but also won't find the price!